frontend/src/App.tsx

```diff
@@ -1,36 +1,93 @@
-import React, {Component} from 'react';
+import React, { Component } from 'react';
+import styled from 'styled-components';
 import logo from './logo.svg';
-import './App.css';
- 
-class App extends Component {
- 
-    state:any = {};
- 
-    componentDidMount() {
-        setInterval(this.hello, 250);
+import LandingPage from './views/Landing';
+import fetchHello from './api/hello';
+
+const StyledApp = styled.div`
+  text-align: center;
+  
+  .App-logo {
+    height: 40vmin;
+    pointer-events: none;
+  }
+  
+  @media (prefers-reduced-motion: no-preference) {
+    .App-logo {
+      animation: App-logo-spin infinite 20s linear;
     }
- 
-    hello = () => {
-        fetch('/hello')
-            .then(response => response.text())
-            .then(message => {
-                this.setState({message: message});
-            });
-    };
- 
-    render() {
-        return (
-            <div className="App">
-                <header className="App-header">
-                    <img src={logo} className="App-logo" alt="logo"/>
-                    <h1 className="App-title">{this.state.message}</h1>
-                </header>
-                <p className="App-intro">
-                    To get started, edit <code>src/App.js</code> and save to reload.
-                </p>
-            </div>
-        );
+  }
+  
+  .App-header {
+    background-color: #282c34;
+    min-height: 100vh;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    font-size: calc(10px + 2vmin);
+    color: white;
+  }
+  
+  .App-link {
+    color: #61dafb;
+  }
+  
+  @keyframes App-logo-spin {
+    from {
+      transform: rotate(0deg);
     }
+    to {
+      transform: rotate(360deg);
+    }
+  }
+`;
+
+type MyProps = {};
+
+type MyState = {
+  message: String,
+};
+
+class App extends Component<MyProps, MyState> {
+
+  constructor(props: MyProps) {
+    super(props);
+
+    this.state = { message: 'Loading' };
+  }
+
+  componentDidMount() {
+    setInterval(this.hello, 250);
+  }
+
+  hello = () => {
+    fetchHello()
+      .then((message) => {
+        this.setState({ message });
+      });
+  };
+
+  render() {
+    const { message } = this.state;
```

Comment:
Why is `message` surrounded by brackets?

Reply:
It's called destructuring according to [this site](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment), basically it'd be equivalent to `const message = this.state.message`. This is also an ESLint suggestion.

Reply:
Ah gotcha, thanks for the explanation.